integrations/kitty/category/Kitty/Plugin/Kitty.hs

```diff
@@ -185,50 +173,24 @@ class SemigroupCat (k :: Type -> Type -> Type) m where
 instance Semigroup m => SemigroupCat (->) m where
   appendK = uncurry (<>)
 
--- | A traced Cartesian category provides a notion of a parameterized fixed-point operator. There is
```

Comment:
These comments seem helpful. Are they no longer applicable? Also why the renaming?

Reply:
It's not a renaming. We used to have both TracedCat and FixedCat, but turns out there's actually TracedCat already in concat-classes, so I just removed our TracedCat, but left our FixedCat in place.

I agree there's useful stuff in the comment (that comment is mostly copied from Haskerwaul, actually), but the part about "Currently, we can't generate @trace@ during conversion, so we don't bother to separate the classes." is false -- we generate it from ArrowLoop, and we do separate the classes (this comment was on FixedCat before we split out TracedCat).

We just don't have a TracedCat to attach the remainder of the comment to.